packages/sit-onyx/src/components/OnyxDataGrid/features/HeaderCell.vue

```diff
@@ -28,6 +28,7 @@ const slots = defineSlots<{
 
   &__actions {
     display: inline-flex;
+    gap: 0.25rem;
```

Comment:
```suggestion
    gap: inherit;
```

Shouldn't the gap be the same as between the header label and the actions?